src/common.ts

```diff
@@ -110,12 +112,13 @@ export class LoggingCommon {
   readonly logName: string;
   private inspectMetadata: boolean;
   private levels: {[name: string]: number};
-  stackdriverLog: Log;
+  stackdriverLog: LogSeverityFunctions;
   private resource: protos.google.api.IMonitoredResource | undefined;
   private serviceContext: ServiceContext | undefined;
   private prefix: string | undefined;
   private labels: object | undefined;
   private defaultCallback?: Callback;
+  redirectToStdout: boolean;
```

Comment:
should it be declared `private` like the rest of variables?

Reply:
I use it same way as stackdriverLog variable, e.g. in middleware.